pkg/apis/kops/instancegroup.go

```diff
@@ -94,6 +94,8 @@ type InstanceGroupSpec struct {
 	CloudLabels map[string]string `json:"cloudLabels,omitempty"`
 	// NodeLabels indicates the kubernetes labels for nodes in this group
 	NodeLabels map[string]string `json:"nodeLabels,omitempty"`
+	// FileAssets is a collection of file assets for this instance group
+	FileAssets []*FileAssetSpec `json:"fileAssets,omitempty"`
```

Comment:
Can we change all the API `[]*FileAssetSpec` to `[]FileAssetSpec` please?  I know we have a lot of `[]*T` but API machinery is happier with `[]T`